tests/components/samsungtv/test_media_player.py

```diff
@@ -138,8 +140,11 @@ def remote_fixture():
 @pytest.fixture(name="wakeonlan")
 def wakeonlan_fixture():
     """Patch the wakeonlan Remote."""
-    with MockDependency("wakeonlan") as wakeonlan:
-        yield wakeonlan
+    with patch(
+        "homeassistant.components.samsungtv.media_player.wakeonlan"
+    ) as wakeonlan_class:
+        wakeonlan_class.return_value.send_magic_packet.return_value = None
```

Comment:
Why do we need to set the return value of that method to `None`?

Reply:
It doesn't matter the value I get from this method, that's why.

I can put a `mock.Mock()` if you prefer.

Reply:
The default is a mock, so we don't need to do anything besides patching the target.

Reply:
Yeppa, done